src/docs/product/product.ts

```diff
@@ -1,71 +1,74 @@
 import { OpenAPIV3 } from 'openapi-types';
 
 const productPaths: OpenAPIV3.PathsObject = {
-  '/api/products': {
+  '/api/products/{searchKeyword}/': {
     get: {
-      summary: 'Get all Products',
+      summary: 'Get products',
       tags: ['Products'],
-      security: [
+      description:
+        'Get product,Search a product by name, category, and price range',
+      parameters: [
         {
-          bearerAuth: [],
-        },
-      ],
-      responses: {
-        '200': {
-          description: 'Find products in Store',
-        },
-        '404': {
-          description: 'No products in store',
+          in: 'path',
```

Comment:
It is better if this search keyword is a query parameter, just like others. Instead of making it an optional path parameter